docs/user-feedback/walkthroughs/strings.md

```diff
@@ -0,0 +1,195 @@
+| `string` support            | Can use the native `string` type they are familiar with | ✅                                  | ❌                     |
+| String literals             | Can use `"` or `'` for literal string values            | ✅                                  | ❌                     |
+| Fully-implemented prototype | IDEs **MAY\*** not show unsupported methods             | ❌                                  | ✅                     |
+| Concatenation with `+`      | Can use `+` to concatenate two strings                  | ✅                                  | ❌                     |
+
+\* TypeScript does have a [plugins feature](https://github.com/microsoft/TypeScript/wiki/Writing-a-Language-Service-Plugin#whats-a-language-service-plugin) that would enable us to modify the IDE experience, but it may be non-trivial for users to set up.
+
+# Code Comparison
+
+### Attempting Regular TypeScript
+
+<table>
+<tr>
+<th>Option 5</th>
+<th>Option 4</th>
+</tr>
+
+<tr>
+<td>
+
+```ts
+getValueAtIndex(input: string, idx: number): string {
+    // Error: [] not supported, use .getByte instead
+    return `The value at ${idx} is ${input[idx]}`;
+}
+
+// The following functions work as-is
+
+helloWorld(): string {
+    return "Hello, World!";
+}
+
+concatStrings(a: string, b: string): string {
+    return a + b;
+}
+```
+
+</td>
+<td>
+
+```ts
+// Error: string not supported, use str instead
+getValueAtIndex(input: string, idx: number): string {
+    // Error: Template literals not supported, use Str tag instead
+    return `The value at ${idx} is ${input[idx]}`;
+}
+
+// Error: string not supported, use str instead
+helloWorld(): string {
+    // Error: String literals not supported, use Str tag instead
+    return "Hello, World!";
+}
+
+// Error: string not supported, use str instead
+concatStrings(a: string, b: string): string {
+    // Error: + not supported on strings, use Str tag instead
+    return a + b;
+}
+```
+
+</td>
+
+</tr>
+</table>
+
+### Algorand TypeScript
+
+<table>
+<tr>
+<th>Option 5: Extended String Prototype</th>
+<th>Option 4: Custom Class</th>
+</tr>
+
+<tr>
+<td>
+
+```ts
+getValueAtIndex(input: string, idx: number): string {
+    // Note use of .getByte instead of []
+    return `The value at ${idx} is ${input.getByte(idx)}`;
+}
+
+helloWorld(): string {
+    return "Hello, World!";
+}
+
+concatStrings(a: string, b: string): string {
+    return a + b;
+}
+
+```
+
+</td>
+<td>
+
+```ts
+getValueAtIndex(input: str, idx: uint64): str {
+    // Instead of string literals, we used a tagged template
+    return Str`The value at ${idx} is ${input[idx]}`;
+}
+
+helloWorld(): string {
+    return Str`Hello, World!`;
+}
+
+concatStrings(a: str, b: str): string {
+    // Instead of using the `+` operator, we used a custom function
+    return concat(a, b);
+}
+```
+
+</td>
+
+</tr>
+</table>
+
+## General Questions
+
+- Would seeing errors in your IDE change your opinion on the proposed solutions?
```

Comment:
Are these questions specifically related to live errors or at compile time?

Reply:
Just clarified this in the recent commit